ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVTest.scala

```diff
@@ -188,6 +188,19 @@ object KVTest {
       testState <- get[KVTestState]
       submInfo = createSubmitterInfo(submitter, commandId, deduplicationTime, testState)
       (tx, txMetaData) = transaction
+      subm = keyValueSubmission.transactionToSubmission(
```

Comment:
Isn't this masked by `subm` in the following statement?

Reply:
Good catch. Thanks.